pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/mappers/SegmentPreprocessingMapper.java

```diff
@@ -94,7 +94,7 @@ public void map(AvroKey<GenericRecord> record, NullWritable value, final Context
       if (!normalizedTimeColumnValue.equals(_sampleNormalizedTimeColumnValue)) {
         // TODO: Create a custom exception and gracefully catch this exception outside, changing what the path to input
         // into segment creation should be
-        LOGGER.error("This segment contains multiple time units. Sample is {}, current is {}", _sampleNormalizedTimeColumnValue, normalizedTimeColumnValue);
+        LOGGER.warn("This segment contains multiple time units. Sample is {}, current is {}", _sampleNormalizedTimeColumnValue, normalizedTimeColumnValue);
```

Comment:
We should avoid to put record level log unless it would really provide the good information. Even if changing the log level to `WARN` won't log to Travis, this will be logged to other jobs (actual hadoop job, TMC test..etc).

Reply:
This will provide good information - see above